common/client.go

```diff
@@ -173,6 +175,9 @@ func (rc *RestClient) modifyUrl(dest string, queryMod url.Values) error {
 // execMethod applies the specified method to the provided url (which is interpreted
 // as relative or absolute).
 func (rc *RestClient) execMethod(method string, dest string, data interface{}, result interface{}) error {
+	// TODO check if token expired, if yes, reauthenticate... But this needs
```

Comment:
If you don't want to change this now, please add a TODO and a new card, describing this issue.